webapp/tests/model/test_eligibility_data.py

```diff
@@ -1162,22 +1050,8 @@ def test_if_other_income_invalid_and_foreign_country_no_then_raise_validation_er
                    MagicMock(side_effect=ValidationError([], OtherIncomeEligibilityData))):
             self.assertRaises(ValidationError, ForeignCountrySuccessEligibility.parse_obj, valid_data)
 
-    def test_if_other_income_valid_and_foreign_country_no_and_no_registration_method_set_then_raise_no_validation_error(self):
-        valid_data = {'foreign_country_eligibility': 'no'}
-        try:
-            with patch('app.model.eligibility_data.OtherIncomeEligibilityData.__init__',
-                       MagicMock(return_value=None)):
-                ForeignCountrySuccessEligibility.parse_obj(valid_data)
-        except ValidationError:
-            self.fail("ForeignCountryEligibility.parse_obj should not raise validation error")
-
-    def test_if_other_income_valid_and_foreign_country_no_but_registration_method_not_software_then_raise_validation_error(self):
-        non_valid_data = {'foreign_country_eligibility': 'no', 'elster_registration_method_eligibility': 'NOT_SOFTWARE'}
-        with patch('app.model.eligibility_data.OtherIncomeEligibilityData.__init__', MagicMock(return_value=None)):
-            self.assertRaises(ValidationError, ForeignCountrySuccessEligibility.parse_obj, non_valid_data)
-
-    def test_if_other_income_valid_and_foreign_country_no_and_registration_method_set_to_software_then_raise_no_validation_error(self):
-        valid_data = {'foreign_country_eligibility': 'no', 'elster_registration_method_eligibility': 'software'}
+    def test_if_other_income_valid_and_foreign_country_no_and_no_user_a_has_elster_account_eligibility_then_raise_no_validation_error(self):
+        valid_data = {'foreign_country_eligibility': 'no', 'user_a_has_elster_account_eligibility': 'no'}
```

Comment:
I think we have to re-evaluate which tests make sense in the new setting and which ones don't. I'm not sure this one soeas while to me a test for `user_a_has_elster_account_eligibility` is missing entirely.